README.md

```diff
@@ -11,6 +11,21 @@ variables. You will need to set F90 (and optionally, F90FLAGS).
 At link time you will need to link with the OpenCL run-time
 libraries on your system.
 
+# Controlling runtime behaviour"
+
+There are 3 environment variables to control the OpenCL execution
+at runtime:
+
+- `FORTCL_KERNELS_FILE`: Specify at run-time the binary or source file
+where the OpenCL kernels are if this is not given at compile-time.
+
+- `FORTCL_PLATFORM`: If a system has multiple OpenCL platforms, this
+environment variable will choose which one to use. If not specified
+it will use platform 1.
+
+- `FORTCL_VERBOSE`: If this environment is set to a value different than
```

Comment:
s/different/other/ Also, please clarify, is this a binary (on/off) switch or does the level of verbosity depend on the value?

Reply:
Done